src/integrationTest/java/uk/gov/hmcts/reform/bulkscanprocessor/tasks/BlobProcessorTaskTest.java

```diff
@@ -139,26 +139,25 @@ private void testBlobFileProcessed() throws Exception {
     @Test
     public void should_process_other_zip_files_if_previous_zip_fails_to_process() throws Exception {
         // given
-        uploadZipToBlobStore("3_24-06-2018-00-00-00.zip"); //Zip with only pdf without metadata
-        uploadZipToBlobStore("7_24-06-2018-00-00-00.zip"); //Zip with pdf and metadata
+        uploadToBlobStorage("bad_24-06-2018-00-00-00.zip", zipDir("zipcontents/missing_metadata"));
+        uploadToBlobStorage("good_24-06-2018-00-00-00.zip", zipDir("zipcontents/ok"));
 
-        Pdf pdf2 = new Pdf("1111002.pdf", toByteArray(getResource("1111002.pdf")));
+        Pdf okPdf = new Pdf("1111002.pdf", toByteArray(getResource("zipcontents/ok/1111002.pdf")));
 
-        given(documentManagementService.uploadDocuments(ImmutableList.of(pdf2)))
+        given(documentManagementService.uploadDocuments(ImmutableList.of(okPdf)))
             .willReturn(ImmutableMap.of(
-                "1111001.pdf", DOCUMENT_URL1,
                 "1111002.pdf", DOCUMENT_URL2
             ));
 
         // when
         processor.processBlobs();
 
         // then
-        // We expect only one envelope 4_24-06-2018-00-00-00.zip which was uploaded
+        // We expect only one envelope from the valid zip file which was uploaded
         Envelope actualEnvelope = getSingleEnvelopeFromDb();
 
         String originalMetaFile = Resources.toString(
-            getResource("metadata_4_24-06-2018-00-00-00.json"),
+            getResource("zipcontents/ok/metadata.json"),
```

Comment:
wrong metadata file was used here (`4...` instead of `7...`, which was uploaded), but those happen to be the same, so test passed...